src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.ts

```diff
@@ -65,6 +67,15 @@ export class DivikComponent implements OnInit {
     return description;
   }
 
+  changeLevel(value) {
+    console.log('change ' + value);
+    if (value > 0 && value <= this.divikConfig.Level) {
+      this.divikService
+        .get(1, 1, value)
```

Comment:
We should use known `preparationId`, shouldn't we? It has been passed through the router.